src/vulkan/device.cc

```diff
@@ -24,6 +25,23 @@ namespace amber {
 namespace vulkan {
 namespace {
 
+const std::vector<const char*> kRequiredValidationLayers = {
+#ifdef __ANDROID__
+    // Note that the order of enabled layers is important. It is
+    // based on Android NDK Vulkan document. Don't change it.
+    "VK_LAYER_GOOGLE_threading",      "VK_LAYER_LUNARG_parameter_validation",
+    "VK_LAYER_LUNARG_object_tracker", "VK_LAYER_LUNARG_core_validation",
+    "VK_LAYER_LUNARG_device_limits",  "VK_LAYER_LUNARG_image",
+    "VK_LAYER_LUNARG_swapchain",      "VK_LAYER_GOOGLE_unique_objects",
+#else   // __ANDROID__
+    // We assume that it runs with LunarG Vulkan SDK if it is not
+    // Android.
+    "VK_LAYER_LUNARG_standard_validation",
```

Comment:
I don't think this is true if you run with MoltenVK.

Reply:
Added a TODO

Reply:
I don't think you want this as a TODO as you're going to break MoltenVK by adding this. You at least should make the else conditional.

Reply:
Based on [doc of MoltenVk](https://github.com/KhronosGroup/MoltenVK/blob/master/Docs/MoltenVK_Runtime_UserGuide.md#known-moltenvk-limitations), we must use validation layers of LunarG.

Let me just update the comment.

Reply:
Have you tested it on Molten? Last time I tried Molten the driver did not report any layers as being available which means we'll always fail.